frontend/src/lib/components/navigation/NavItem.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+  import {error} from '@sveltejs/kit';
```

Comment:
Is this required anymore? See the Github warning

Reply:
True! I saw the warning as well and forgot to update this :D It's now corrected. I also improved the documentation a tiny bit.